helm/config/backend.yaml

```diff
@@ -76,10 +76,9 @@ modelsources:
     username: sampleUser
     password: samplePassword
 
-    host: "{{ .Values.t4cServer.host }}"
-    port: "{{ .Values.t4cServer.port }}"
-    cdoPort: "{{ .Values.t4cServer.cdoPort }}"
-    licence: "{{ .Values.t4cServer.licenseKey }}"
+    host: localhost
+    port: 2036
+    licence: test
```

Comment:
The whole section `modelsources` should not be used anymore in `v2`. I will remove it in another branch. 
There all still some old references in the backend to the `modelsources` configuration, but they will be removed with #221.

Reply:
Okay.

Reply:
Will be handled in https://github.com/DSD-DBS/capella-collab-manager/pull/261.